### ISSUE TITLE
Extra blank lines after imports are removed

### ISSUE THREAD

**Issue by nyuszika7h** [lyz-code/autoimport#219]
## Description
autoimport removes extra blank lines after imports, even when it’s required for PEP 8 compliance (e.g. when imports are directly followed by a `class` or a `def`). And also I like doing `lines_after_imports = 2` in isort for consistency anyway.

While you may say I can just run another auto-formatter afterwards to fix the issue, this actually makes it impossible to use autoimport as a pre-commit hook, as on the next run autoimport will remove the blank line again, and pre-commit requires all hooks to succeed.

## Steps to reproduce
1. Create a file with the following contents:
   ```py
    import random


    def foo():
        print(random.random())
    ```
2. Run `autoimport foo.py`

## Current behavior
The file gets rewritten to this:
```py
    import random

    def foo():
        print(random.random())
```

## Desired behavior
The file should remain untouched.

## Environment
```
           autoimport version: 1.2.2
               python version: 3.10.4 (main, Apr  4 2022, 14:54:08) [GCC 10.2.1 20210110]
                     platform: Linux-5.10.0-16-amd64-x86_64-with-glibc2.31
```

**Comment by lyz-code** [comment]
Hi @nyuszika7h thanks for taking the time to open the issue. I see your point and agree it should be fixed.

Sadly I don't have time to implement this on my own, but if anyone is interested in contributing I think the easier way to do it is by implementing some logic when [the different parts of the code are joined](https://github.com/lyz-code/autoimport/blob/main/src/autoimport/model.py#L199) so that if the `self.code` starts with `def` or `class` it should add a new `\n`

**Comment by lyz-code** [comment]
Available since `1.3.1`
